magento/pwa-studio
# Scroll top when payment information is done

## Description

It was unclear what happens when a user clicks "Review Order". The page stays locked to its scroll position but the dom changes when the items component renders and the payment done card is displayed. The desired effect is to scroll the page once the payment information section is done.

## Related Issue





Closes [PWA-697](https://jira.corp.magento.com/browse/PWA-697).

## Acceptance 



### Verification Stakeholders

### Specification



### Verification Steps


1. Go to checkout and proceed through steps up to payment info.
2. Click Review Order after filling payment info.
3. See that page scrolls to top.

## Screenshots / Screen Captures (if appropriate)

[![Image from Gyazo](https://i.gyazo.com/a179b0f574bb2c6864c807241b62fdd3.gif)](https://gyazo.com/a179b0f574bb2c6864c807241b62fdd3)

## Checklist

* I have updated the documentation accordingly, if necessary.
